### DISCUSSION TITLE
电脑传输2G文件到手机，突然就完成自己闪退，手机上显示传输完成但是没有文件，显示错误

### DISCUSSION THREAD

**Discussion by Timothy-att** [localsend/localsend#1514] [L-简体中文]
![IMG_0848](https://github.com/user-attachments/assets/358b74ed-5cf9-4bed-af9f-a1aa45f38263)

**Comment by Buhaoran0307** [comment]
一样的问题，帮顶

**Comment by 7r1UMPH** [comment]
Hi, I also encountered this problem, but it happened when transferring files from Android to Windows. I successfully solved this problem by changing
1. the default save directory of LocalSend on Windows and Android
2. using the automatic repair function in the troubleshooting function on Windows
. I don't know what specific operation solved this problem. I hope it helps you!
